io.enpass.Enpass.yaml

```diff
@@ -0,0 +1,101 @@
+# io.enpass.Enpass.yaml
+app-id: io.enpass.Enpass
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: enpass
+tags: [proprietary]
+separate-locales: false
+finish-args:
+    # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+    # Notification access
+  - --talk-name=org.freedesktop.Notifications
+    # System Tray Icon
+  - --talk-name=com.kde.StatusNotifierWatcher
+    # Secret Service
+  - --own-name=org.freedesktop.secrets
+    # Favicon download
+  - --share=network
+    # Required to save vaults in default location <$HOME/Documents>
+  - --filesystem=xdg-documents
+modules:
+  - name: enpass
+    buildsystem: simple
+    build-options:
+      env:
+        FLATPAK_ID: io.enpass.Enpass
+    build-commands:
+      - |
+
+        mkdir -p ${FLATPAK_DEST}/bin/extra
+        mkdir -p ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128
+        mkdir -p ${FLATPAK_DEST}/share/mime/packages
+
+        install -Dm 755 apply_extra --target-directory ${FLATPAK_DEST}/bin/
+        chmod +x ${FLATPAK_DEST}/bin/apply_extra
+
+        install -Dm 755 enpass --target-directory ${FLATPAK_DEST}/bin/
+        chmod +x ${FLATPAK_DEST}/bin/enpass
```

Comment:
```suggestion
```

Reply:
Understood.

I've removed the redundant commands in [this commit](https://github.com/ShobuPrime/flathub/commit/9cbe7f468a0fcedc068dfc60c55cc58219503d44?diff=split) :heavy_check_mark: